soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -148,6 +148,7 @@ config NRF53_SYNC_RTC_INIT_PRIORITY
 	  nRF53 Synchronized RTC initialization priority.
 
 config NRF_RTC_TIMER_USER_CHAN_COUNT
+	default 3 if NRF_802154_RADIO_DRIVER
```

Comment:
If we need three channels for nRF 802.15.4 Radio Driver and one more for sync RTC: https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/nordic_nrf/nrf53/sync_rtc.c#L244, then we should have `default 4` in this line, right?

Reply:
sync_rtc allocates and uses the channel only during initialization, then frees it. The 802154 driver can then allocate this channel when an operation using hw-task is requested. So `default 3` is enough.

Reply:
Fair enough.

Reply:
Perhaps a comment with the above explanation would be nice here to avoid misunderstandings in the future.